### DISCUSSION TITLE
1.5.2.1

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#589] [Обновления]
## Список изменений
- Фикс списка `@connect`

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.5.2...1.5.2.1

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.5.2.1'>1.5.2.1</a>.</em>

**Comment by S-Bashik** [comment]
Здравствуете.
Если вчера с утра работало, то сегодня нет. После обновление перестало работать.
Даже собственный интерфейс теперь не переводит.
Раньше можно было сменить домены, то теперь, не подходит не один.
[Скриншот (Firefox)](https://i.imgur.com/bOyl5FC.jpeg)
[Скриншот (Firefox)](https://i.imgur.com/KIXivNT.jpeg)
[Скриншот](https://i.imgur.com/SIU8Fwl.jpeg)
[Скриншот (Opera)](https://i.imgur.com/sAq1EUt.jpeg)

**Reply by NullVerdict** [reply]
Ну как вариант попробуй поменять все connect в скрипте на ```// @connect        *```

Измени 

```
// @connect        yandex.net
// @connect        raw.githubusercontent.com
// @connect        toil.cc
// @connect        deno.dev
// @connect        onrender.com
// @connect        workers.dev
```

на

```
// @connect        *
````

и разреши для всех доменов

**Comment by adetyantsev** [comment]
Привет, при нажатии на кнопку перевести видео в тик-ток пишет такую ошибку "Возникла ошибка при переводу" https://imgur.com/1gDQLpp. Подскажите с чем может быть связано?

**Comment by S-Bashik** [comment]
> Ну как вариант попробуй поменять все connect в скрипте на `// @connect        *`

Ведь не все понимают, о чём речь, и вряд ли простой юзер полезет что-то менять.

В  моём случае это бесполезно, перепробовал варианты, но безрезультатно, не переводит... да интерфейс всё также без изменений.
Возможно из-за этого... хотя, кто его знает, **у всех всё разное**.
> и разреши для всех доменов 

[Скрин](https://i.imgur.com/mPFCaeL.jpeg)
Я не знаю как в Adguard это сделать, ведь носитель расширения у меня не браузер, а программа "Adguard".

Единственное, что мне помогло, так это установить сохранённую версию "vot-cloudflare.user (1.5.1.5)"... [Скрин](https://i.imgur.com/QdHevFe.jpeg)
И всё нормализовалось... [Скрин](https://i.imgur.com/AO9UCHH.jpeg)
Но опять же, через время, расширение **снова обновится до последней версии**. Так как в Adguard я не нашёл как запретить обновления по раздельности дополнений. [Скрин](https://i.imgur.com/9jrYcft.jpeg)

Пришлось переписывать версию в скрипте с  **"1.5.1.5" на "1.5.2.1"**, и это вроде пока помогло, скрипт не обновляется.
[Скрин](https://i.imgur.com/vxv2x3U.jpeg)
[Скрин](https://i.imgur.com/J8pLalp.jpeg)

З.Ы.
**Но ведь это не совсем решение... и проблема вернётся если не исправят.**
Как вариант, вернуть как было, когда всё работало...

**Reply by NullVerdict** [reply]
Можешь посмотреть ошибки в консоли?

**Reply by S-Bashik** [reply]
Где находится эта консоль? В настройках Adguard? Подобного там не увидел.
[Скриншот](https://i.imgur.com/CjL50Gl.jpeg)

Или в консоли браузера? Так у меня во всех браузерах проблема с переводом появилась...

**Reply by ilyhalight** [reply]
Скорее всего это из-за GM_fetch (GM_xmlHttpRequest) такое поведение. Но все равно нужно посмотреть логи из консоли браузера

**Reply by S-Bashik** [reply]
Пришлось снова установить багованную версию "1.5.2.1", даже в браузере профиль убрал на время, дабы браузер был как с нуля (из коробки).
Вот что получилось...
[Скриншот](https://i.imgur.com/dydELcP.jpeg)

А это я уже снова вернулся на сохранённую у себя версию "1.5.1.5", даже не одного упоминания в консоли о "[VOT] [localizationProvider] locale"... не знаю, так должно быть?
[Скриншот](https://i.imgur.com/X5toI2Q.jpeg)

**Reply by ilyhalight** [reply]
Возможно исправил - https://drive.google.com/drive/folders/1tUGJQV9iqzx_iSSN1PsKyg2tMcmZWd5H?usp=sharing

**Reply by S-Bashik** [reply]
> Возможно исправил...

Благодарю, исправили.
[Скриншот](https://i.imgur.com/ajcez9p.jpeg)

**Comment by shakhop** [comment]
hello i would like to ask why it doesn’t translate to italian? and how can i fix it?

**Reply by ilyhalight** [reply]
![изображение](https://github.com/ilyhalight/voice-over-translation/assets/62353659/96f789e5-1955-47da-92a1-44487dc4bb42)

**Reply by shakhop** [reply]
:(

**Reply by shakhop** [reply]
Thank u anyway u r great
